### PR TITLE
Enable thread priority for Windows

### PR DESCRIPTION
**Pull request by MichaelOrlov**
- Relates https://github.com/ros2/rosbag2/pull/1457

### CODE REVIEW

**Comment by MichaelOrlov** [comment]
@jmachowinski Friendly ping for review and merge of this PR if you have no concerns or questions about it.

File: rosbag2_compression/src/rosbag2_compression/sequential_compression_writer.cpp [hunk 1]
```diff
@@ -71,39 +71,38 @@ void SequentialCompressionWriter::compression_thread_fn()
...
-
 #else
     int wanted_nice_value = *compression_options_.thread_priority;
 
     errno = 0;
     int cur_nice_value = getpriority(PRIO_PROCESS, 0);
     if (cur_nice_value == -1 && errno != 0) {
       ROSBAG2_COMPRESSION_LOG_WARN_STREAM(
-        "Could not set nice value of compression thread to " << wanted_nice_value <<
+        "Could not set nice value of compression thread to the: " << wanted_nice_value <<
```
Comment (jmachowinski) [inline rosbag2_compression/src/rosbag2_compression/sequential_compression_writer.cpp]: either remove the 'the', or change it to 
"Could not set nice value of compression thread to the wanted_nice_value: " << wanted_nice_value <<

**Review by jmachowinski — COMMENTED** [review]
Rest looks fine

File: rosbag2_compression/src/rosbag2_compression/sequential_compression_writer.cpp (hunk 1 above)
Comment (MichaelOrlov) [reply rosbag2_compression/src/rosbag2_compression/sequential_compression_writer.cpp]: @jmachowinski Thanks for notice. I've removed `"the"` from the log statements.

**Pull request merged**